.circleci/config.yml

```diff
@@ -0,0 +1,80 @@
+defaults: &defaults
+  docker:
+    - image: circleci/node:8.10
```

Comment:
`iopipe/circleci` here

Reply:
Oh good call, I wasn't thinking of the acceptance test needs